orchagent/crmorch.cpp

```diff
@@ -65,6 +67,8 @@ const map<CrmResourceType, uint32_t> crmResSaiAvailAttrMap =
     { CrmResourceType::CRM_DNAT_ENTRY, SAI_SWITCH_ATTR_AVAILABLE_DNAT_ENTRY },
     { CrmResourceType::CRM_MPLS_INSEG, SAI_OBJECT_TYPE_INSEG_ENTRY },
     { CrmResourceType::CRM_MPLS_NEXTHOP, SAI_SWITCH_ATTR_AVAILABLE_IPV4_NEXTHOP_ENTRY },
+    { CrmResourceType::CRM_SRV6_MY_SID_ENTRY, SAI_OBJECT_TYPE_MY_SID_ENTRY },
+    { CrmResourceType::CRM_SRV6_NEXTHOP, SAI_OBJECT_TYPE_NEXT_HOP },
```

Comment:
isn't this SAI_SWITCH_ATTR_AVAILABLE_IPV4_NEXTHOP_ENTRY. @smaheshm has recently fixed the MPLS_NEXTHOP above

Reply:
Changed to SAI_SWITCH_ATTR_AVAILABLE_IPV6_NEXTHOP_ENTRY